docs/asyncapi.yaml

```diff
@@ -130,8 +130,9 @@ components:
       example: 1234567890
     Filename:
       type: string
-      description: PDF filename
-      example: FFC_PaymentStatement_SFI_2022_1234567890_2022080515300000.pdf
+      description: PDF filename, formatted as product code using 3 to 6 uppercase characters, descriptor in PascalCase, Scheme ID using 3 to 6 uppercase characters, Scheme Year using 4 digits, FRN using 10 digits, date and time using 16 digits and then .pdf. All separated by underscore.
+      example: FFC_PaymentStatement_SFI_2022_1234567890_202208051530.pdf
```

Comment:
The example does not match the reg-exp, timestamp length in reg-exp is 16 but 12 in example.

Reply:
Updated example